package.json

```diff
@@ -140,7 +143,7 @@
               },
               "mode": {
                 "type": "string",
-                "description": "One of 'debug', 'test', 'exec'.",
+                "description": "One of 'debug', 'debug_remote', 'test', 'exec'.",
```

Comment:
I'd lean toward using just "remote" here.  The previous names in this list were all aligned with arguments to `dlv`, but this one won't be since it won't launch `dlv` itself.  In principle, it could be used to do the remote equivalent of any of the other modes, depending on what was used on the server.